core/src/test/scala/integration/kafka/api/CustomQuotaCallbackTest.scala

```diff
@@ -136,9 +148,11 @@ class CustomQuotaCallbackTest extends IntegrationTestHarness with SaslSetup {
     // Create large number of partitions on another broker, should result in throttling on first partition
     val largeTopic = "group1_largeTopic"
     createTopic(largeTopic, numPartitions = 99, leader = 0)
+    user.removeThrottleMetrics()
```

Comment:
this is unnecessary if we remove the unnecessary call of `updateClusterMetadata`